<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArduMYControllerWidget</class>
 <widget class="QWidget" name="ArduMYControllerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="TryToggle" name="tryToggleConnect" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSync">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Sync</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tabStatus">
      <attribute name="title">
       <string>Status</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="4">
        <widget class="QLabel" name="labelIntroduction">
         <property name="text">
          <string>&lt;table width=&quot;100%&quot;&gt;
&lt;tr&gt;
&lt;td&gt;
&lt;h1&gt;ArduMY™ Controller version 1.0&lt;/h1&gt;
&lt;p&gt;This is the native Arduino© based controller for OctoMY™&lt;/p&gt;
&lt;p&gt;You will find full documentation for it &lt;a href=&quot;http://www.octomy.org/documentation/development/architectual-overview/ardumy&quot; &gt;here&lt;/a&gt;&lt;/p&gt;
&lt;/td&gt;
&lt;td&gt;
&lt;img src=&quot;qrc:/images/ardumy_badge.svg&quot; alt=&quot;ArduMY™ Badge&quot; style=&quot;float:right; display:block;&quot;/&gt;
&lt;/td&gt;
&lt;/tr&gt;
&lt;/table&gt;</string>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QLabel" name="labelQuickGuide">
         <property name="text">
          <string>&lt;h2&gt;Quick guide&lt;/h2&gt;
&lt;ol&gt;
&lt;li&gt;Get a &lt;a href=&quot;http://www.octomy.org/documentation/development/architectual-overview/ardumy&quot; &gt;compatible&lt;/a&gt; Arduino© board.&lt;/li&gt;
&lt;li&gt;Go &lt;a href=&quot;http://www.octomy.org/documentation/development/architectual-overview/ardumy&quot; &gt;download&lt;/a&gt;, compile and upload latest version of ArduMY™ on your board .&lt;/li&gt;
&lt;li&gt;Connect your board to this Agent via USB/Serial cable or pair with Bluetooth.&lt;/li&gt;
&lt;li&gt;Go the Serial tab and configure serial parameters. NOTE: The green serial status light on general tab should light up when it is working.&lt;/li&gt;
&lt;li&gt;Go the Actuators tab and configure the actuatuators you will be using for your robot. Remember to give them meaningful names.&lt;/li&gt;
&lt;li&gt;You are now ready to start using this agent!&lt;/li&gt;
&lt;/ol&gt;</string>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QPushButton" name="pushButtonShowQuickGuide">
         <property name="text">
          <string>Show Quick Guide</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>292</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Board:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="labelBoardType">
         <property name="text">
          <string>N/A</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>292</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Version:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="labelBoardVersion">
         <property name="text">
          <string>N/A</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="4">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>211</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
      <zorder>labelBoardType</zorder>
      <zorder>label_4</zorder>
      <zorder>labelBoardVersion</zorder>
      <zorder>labelIntroduction</zorder>
      <zorder>label_2</zorder>
      <zorder>pushButtonShowQuickGuide</zorder>
      <zorder>labelQuickGuide</zorder>
     </widget>
     <widget class="QWidget" name="tabSerial">
      <attribute name="title">
       <string>Serial</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="SerialSettingsWidget" name="widgetSerialSettings" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabActuators">
      <attribute name="title">
       <string>Actuators</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="ArduMYActuatorManagementWidget" name="widgetActuatorManager" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Add Actuator</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxAddActuator">
           <item>
            <property name="text">
             <string>Select type...</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>RC Servo</string>
            </property>
            <property name="icon">
             <iconset resource="../../libres/resources/icons.qrc">
              <normaloff>:/icons/rc_servo.svg</normaloff>:/icons/rc_servo.svg</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Step Motor</string>
            </property>
            <property name="icon">
             <iconset resource="../../libres/resources/icons.qrc">
              <normaloff>:/icons/stepper_motor.svg</normaloff>:/icons/stepper_motor.svg</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DC Motor</string>
            </property>
            <property name="icon">
             <iconset resource="../../libres/resources/icons.qrc">
              <normaloff>:/icons/dc_motor.svg</normaloff>:/icons/dc_motor.svg</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Relay Switch</string>
            </property>
            <property name="icon">
             <iconset resource="../../libres/resources/icons.qrc">
              <normaloff>:/icons/relay.svg</normaloff>:/icons/relay.svg</iconset>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTest">
      <attribute name="title">
       <string>Test</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>479</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="ArduinoPinSelector" name="widgetPinThing_1" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ArduinoPinSelector" name="widgetPinThing_2" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ArduinoPinSelector" name="widgetPinThing_3" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ArduinoPinSelector" name="widgetPinThing_4" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ArduinoPinSelector" name="widgetPinThing_5" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ArduinoPinSelector" name="widgetPinThing_6" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>479</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>485</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TryToggle</class>
   <extends>QWidget</extends>
   <header location="global">components/TryToggle.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SerialSettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">hardware/serial/SerialSettingsWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ArduinoPinSelector</class>
   <extends>QWidget</extends>
   <header location="global">ardumy/arduino/ArduinoPinSelector.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ArduMYActuatorManagementWidget</class>
   <extends>QWidget</extends>
   <header location="global">ardumy/ArduMYActuatorManagementWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../libres/resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
